fizzbuzz.js

```diff
@@ -1,9 +1,31 @@
 // This is our main function
 function fizzbuzz() {
-    console.log("Hello, World!");
 
-    // Put your code here...
+    
 
+    for (let n = 0; n <= 255; n++) {
+        let fizzyArray = [];
+        let fizzyOutput = '';
+
+        if (isMult(n, 3)) { fizzyArray.push('Fizz'); }
+        if (isMult(n, 5)) { fizzyArray.push('Buzz'); }
+        if (isMult(n, 7)) { fizzyArray.push('Bang'); }
+        if (isMult(n, 11)) { fizzyArray = ['Bong']; }
+        if (isMult(n, 13)) { 
+            const firstBIndex = fizzyArray.findIndex(word => /^B/.test(word));
+            firstBIndex === -1 ? fizzyArray.push('Fezz') : fizzyArray.splice(firstBIndex,0,'Fezz');
+        }
+        if (isMult(n, 17)) { fizzyArray.reverse() }
+
+        fizzyOutput = fizzyArray.length > 0 ? fizzyArray.join('') : n;
+
+        console.log(fizzyOutput);
+    }
+
+}
+
+const isMult = (n,m) => {
```

Comment:
nice work separating this into a function!
we can use more descriptive names to make it more clear what the function is doing & what each parameter is
```suggestion
const isMultipleOf = (divisor, dividend) => {
```